bin/server.ts

```diff
@@ -26,7 +27,6 @@ void container.load().then(container => {
       extended: true,
     }))
     app.use(bodyParser.json())
-    app.use(bodyParser.urlencoded({ extended: true }))
```

Comment:
removed this as a duplicate of line 26
this was originally copied from auth, so perhaps I should remove it from there too?

it would still work if it was removed altogether, because AFAIU this only handles `application/x-www-form-urlencoded` content types (http://expressjs.com/en/resources/middleware/body-parser.html#bodyparserurlencodedoptions) and we are more interested in JSON

Reply:
Ah ok, my bad 👍